nixnet/_session/base.py

```diff
@@ -335,6 +335,37 @@ def disconnect_terminals(self, source, destination):
         """
         _funcs.nx_disconnect_terminals(self._handle, source, destination)
 
+    def write_state_lin_schedule_change(self, state_value):
+        # type: () -> None
+        """Writes communication states of an XNET session.
+
+        This function writes a request for the LIN interface to change
+        the running schedule.
+
+        According to the LIN protocol, only the master executes schedules,
+        not slaves. If the
+        :any:`nixnet._session.intf.Interface.lin_master` property is false (slave),
+        this write function implicitly sets that property to true (master). If the
+        interface currently is running as a slave, this write returns an error,
+        because it cannot change to master while running.
+
+        Args:
+            state_value(str): Writes the desired state.
+        """
+        _funcs.nx_write_state(self._handle, constants.WriteState.LIN_SCHEDULE_CHANGE, state_value)
+
+    def write_state_lin_diagnostic_schedule_change(self, state_value):
+        # type: () -> None
+        """Writes communication states of an XNET session.
+
+        This function writes a request for the LIN interface to change
+        the diagnostic schedule.
+
+        Args:
+            state_value(str): Writes the desired state.
```

Comment:
Again, I think we can do better about the parameter name.  One reason this is important is the parameter names are part of the API definition and changing them breaks compatibility

For example, you can do
```python
session.write_state_lin_diagnostic_schedule_change(state_value=constants.NULL)
```

From reading the documentation, my recommendation
```
schedule(:any:`nixnet._enums.LinDiagnosticSchedule`):  diagnostic schedule that the LIN master executes